app/addons/databases/actions.js

```diff
@@ -0,0 +1,123 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+define([
+  'app',
+  'api',
+  'addons/databases/stores',
+  'addons/databases/actiontypes',
+  'addons/databases/resources'
+],
+function (app, FauxtonAPI, Stores, ActionTypes, Resources) {
+  return {
+
+    init: function (databases) {
+      var params = app.getParams();
+      var page = params.page ? parseInt(params.page, 10) : 1;
+      var perPage = FauxtonAPI.constants.MISC.DEFAULT_PAGE_SIZE;
+
+      this.setStartLoading();
+      FauxtonAPI.when(databases.fetch({ cache: false })).then(function () {
+        FauxtonAPI.when(databases.paginated(page, perPage).map(function (database) {
+          return database.status.fetchOnce();
+        })).always(function () {
+          //make this always so that even if a user is not allowed access to a database
+          //they will still see a list of all databases
+          FauxtonAPI.dispatch({
+            type: ActionTypes.DATABASES_INIT,
+            options: {
+              collection: databases.paginated(page, perPage),
+              backboneCollection: databases,
+              page: page
+            }
+          });
+        }.bind(this));
+      }.bind(this));
+    },
+
+    setPage: function (_page) {
+      FauxtonAPI.dispatch({
```

Comment:
Small thing. We don't use underscore prefixes for local vars. Mind just calling it `page`?